app/uk/gov/hmrc/bindingtariffclassification/model/Decision.scala

```diff
@@ -33,6 +35,5 @@ case class Decision
 
 case class Appeal
```

Comment:
Created this as an object to save us any migration headaches when we inevitably need to add more fields onto an Appeal